src/main/java/uk/gov/hmcts/reform/sscs/service/FurtherEvidenceService.java

```diff
@@ -1,68 +1,107 @@
-import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
 import uk.gov.hmcts.reform.sscs.docmosis.domain.Pdf;
-import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.domain.FurtherEvidenceLetterType;
 
 @Service
 public class FurtherEvidenceService {
 
     @Autowired
     private CoverLetterService coverLetterService;
+
     @Autowired
     private SscsDocumentService sscsDocumentService;
+
     @Autowired
     private BulkPrintService bulkPrintService;
-    @Autowired
-    private CcdService ccdService;
-    @Autowired
-    private IdamService idamService;
 
+    private final String furtherEvidenceOriginalSenderTemplateName;
+
+    private final String furtherEvidenceOtherPartiesTemplateName;
+
+    public FurtherEvidenceService(@Value("${docmosis.template.609-97.name}") String furtherEvidenceOriginalSenderTemplateName,
+                                  @Value("${docmosis.template.609-98.name}") String furtherEvidenceOtherPartiesTemplateName,
+                                  @Autowired CoverLetterService coverLetterService,
+                                  @Autowired SscsDocumentService sscsDocumentService,
+                                  @Autowired BulkPrintService bulkPrintService) {
+        this.furtherEvidenceOriginalSenderTemplateName = furtherEvidenceOriginalSenderTemplateName;
+        this.furtherEvidenceOtherPartiesTemplateName = furtherEvidenceOtherPartiesTemplateName;
+        this.coverLetterService = coverLetterService;
+        this.sscsDocumentService = sscsDocumentService;
+        this.bulkPrintService = bulkPrintService;
+
+    }
 
     public void issue(SscsCaseData caseData, DocumentType documentType) {
-        List<Pdf> pdfsToBulkPrint = buildPdfsToBulkPrint(caseData, documentType);
-        bulkPrintService.sendToBulkPrint(pdfsToBulkPrint, caseData);
-        setEvidenceIssuedFlagToYes(caseData, documentType);
+        List<Pdf> pdfs = sscsDocumentService.getPdfsForGivenDocType(caseData.getSscsDocument(), documentType);
+
+        send609_97_OriginalSender(caseData, documentType, pdfs);
+        send609_98_OtherParty(caseData, documentType, pdfs);
+        send609_98_Dwp(caseData, pdfs);
     }
 
-    private void setEvidenceIssuedFlagToYes(SscsCaseData caseData, DocumentType documentType) {
-        sscsDocumentService.filterByDocTypeAndApplyAction(caseData.getSscsDocument(), documentType,
-            doc -> doc.getValue().setEvidenceIssued("Yes"));
-        ccdService.updateCase(
-            caseData,
-            Long.valueOf(caseData.getCcdCaseId()),
-            EventType.UPDATE_CASE_ONLY.getCcdType(),
-            "Update case data only",
-            "Simply update case data with no callbacks afterwards",
-            idamService.getIdamTokens());
+    private void send609_97_OriginalSender(SscsCaseData caseData, DocumentType documentType, List<Pdf> pdfs) {
+        byte[] bulkPrintList60997 = buildPdfsFor609_97(caseData, findLetterType(documentType));
```

Comment:
consistent name convention. keep using 609_97 (if thats is what it is)

Reply:
check style forced me to remove underscore from variable name